plugin/core/sessions.py

```diff
@@ -1338,6 +1338,9 @@ def session_buffers_async(self) -> Generator[SessionBufferProtocol, None, None]:
         """
         yield from self._session_buffers
 
+    def get_session_buffer_by_id(self, buffer_id: int) -> SessionBufferProtocol:
```

Comment:
This should be typed `Optional[SessionBufferProtocol]` if it can return `None` for some unknown id.

Reply:
This line cause code actions unavailable when file doesn't have any change.

![image](https://github.com/sublimelsp/LSP/assets/51814639/4a455cb1-0dc9-4aa1-a5b8-9488a13fa0fd)